deps/deps.go

```diff
@@ -254,6 +258,21 @@ func (deps *Deps) PopulateRemainingDeps(ctx context.Context, cctx *cli.Context,
 		}()
 	}
 
+	if deps.EthClient == nil {
+		deps.EthClient = lazy.MakeLazy[*ethclient.Client](func() (*ethclient.Client, error) {
+			// todo: this is a hack, just use the lotus chain api client above
+			switch build.BuildType {
+			case build.BuildCalibnet:
+				return ethclient.Dial("https://api.calibration.node.glif.io/rpc/v1")
+			case build.BuildMainnet:
+				return ethclient.Dial("https://api.node.glif.io/rpc/v1")
+			default:
+				panic("fevm rpc url unknown for this network")
+			}
```

Comment:
Non-blocking: Still definitely a hack

Reply:
We can use the full node for this instead of depending on the glif nodes.